google/cloud/bigtable/data_client.cc

```diff
@@ -139,6 +139,14 @@ class DefaultDataClient : public DataClient {
       ::grpc::CompletionQueue* cq, void* tag) override {
     return impl_.Stub()->AsyncSampleRowKeys(context, request, cq, tag);
   }
+  std::unique_ptr<::grpc::ClientAsyncReaderInterface<
```

Comment:
We talked about this over IM too, but consider returning something like `std::unique_ptr<google::cloud::NewClass<::google::bigtable::v2::SampleRowKeysResponse>>` here.